src/marker.c

```diff
@@ -761,7 +761,6 @@ verify_bytepos (ptrdiff_t charpos)
 void
 syms_of_marker (void)
 {
-  defsubr (&Smarker_position);
```

Comment:
Add `pub use markers::Fmarker_position;` to `src/lib.rs`, then you can remove `C` defun